lib/emulator-manager.js

```diff
@@ -31,16 +31,22 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.killEmulator = exports.launchEmulator = void 0;
 const exec = __importStar(require("@actions/exec"));
 const EMULATOR_BOOT_TIMEOUT_SECONDS = 600;
+let ENABLE_LOGCAT = false;
 /**
  * Creates and launches a new AVD instance with the specified configurations.
  */
-function launchEmulator(apiLevel, target, arch, profile, sdcardPathOrSize, avdName, emulatorOptions, disableAnimations) {
+function launchEmulator(apiLevel, target, arch, profile, sdcardPathOrSize, avdName, emulatorOptions, disableAnimations, disableSpellChecker, disableAutofill, longPressTimeout, enableHwKeyboard, enableLogcat) {
     return __awaiter(this, void 0, void 0, function* () {
         // create a new AVD
         const profileOption = profile.trim() !== '' ? `--device '${profile}'` : '';
         const sdcardPathOrSizeOption = sdcardPathOrSize.trim() !== '' ? `--sdcard '${sdcardPathOrSize}'` : '';
         console.log(`Creating AVD.`);
         yield exec.exec(`sh -c \\"echo no | avdmanager create avd --force -n "${avdName}" --abi '${target}/${arch}' --package 'system-images;android-${apiLevel};${target};${arch}' ${profileOption} ${sdcardPathOrSizeOption}"`);
+        // set number of cores to 2 (instead of 1 because there are 3 cores on the Github-hoster macOS VM)
+        yield exec.exec(`sh -c \\"printf 'hw.cpu.ncore=2\n' >> ~/.android/avd/"${avdName}".avd"/config.ini`);
```

Comment:
Same as below: https://github.com/Doist/android-emulator-runner/pull/1/files#r582439112

Reply:
Actually, I saw this from the PR description _ The lib folder is generated by npm._ after posting the comment so it's probably not applicable. 😅